src/analysis/normed_space/complemented.lean

```diff
@@ -0,0 +1,95 @@
+/-
+Copyright (c) 2020 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Yury Kudryashov
+-/
+import analysis.normed_space.banach
+import analysis.normed_space.finite_dimension
+
+/-!
+# Complemented subspaces of normed vector spaces
+
+A submodule `p` of a topological module `E` over `R` is called *complemented* if there exists
+a continuous linear projection `f : E →ₗ[R] p`, `∀ x : p, f x = x`. We prove that for
+a subspace of a normed space this condition is equivalent to existence of a closed subspace `q`
```

Comment:
```suggestion
a closed subspace of a normed space this condition is equivalent to existence of a closed subspace `q`
```
Also, I could not find the statement of this equivalence below, could you add it (in the form: complemented `p` iff `is_closed p` and `exists q, is_closed q and is_compl p q`)?

Reply:
I just wondered why you proved this for subspaces of normed spaces: isn't all this discussion true over any topological ring?

Reply:
What should I use instead of Banach open mapping theorem in `prod_equiv_of_closed_compl`?

Reply:
Yep, you're right, I don't see how to avoid this.